packages/math-input/src/components/keypad/keypad-page-items.tsx

```diff
@@ -43,6 +43,7 @@ export const KeypadButton = ({
     style,
 }: KeypadButtonProps): React.ReactElement => (
     <Button
+        keyConfig={keyConfig}
```

Comment:
**suggestion**: It looks like all we use in the Button is the button ID.   I'd be inclined to just pass that in instead of the entire `keyConfig`.